File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_savemodel.py
# Specify a model and the file path where it is to be saved. If no path is specified, the model will be saved to the current working directory
model_path = h2o.save_model(
        model = model, 
        #path = "/tmp/mymodel",
        force = True)

print model_path
# /tmp/mymodel/DeepLearning_model_python_1441838096933

File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_savemodel.R
# Specify a model and the file path where it is to be saved. If no path is specified, the model will be saved to the current working directory
model_path <- h2o.saveModel(object = model,
                            path=getwd(), force = TRUE)

print(model_path)
# /tmp/mymodel/DeepLearning_model_R_1441838096933
